src/ray/common/scheduling/scheduling_ids.cc

```diff
@@ -0,0 +1,50 @@
+#include "scheduling_ids.h"
+
+int64_t ScheduleIds::getIdByInt(string sid) {
+  if (string_to_int.count(sid) == 0) return 0;
```

Comment:
should be `-1` or some other value that the hasher cannot produce. Also, the pattern should be
```
auto it = string_to_int.find(sid);
if (it == string_to_int.end()) {
  return -1;
} else {
  return *it;
}
```
(otherwise, each call will do two hash table lookups).

Reply:
done. Note it should be return it->second